_rules/link-accessible-name-c487ae.md

```diff
@@ -153,7 +154,7 @@ When `link` is off screen.
 
 #### Passed Example 10
 
-`area` element with `href` attribute has [accessible name][].
+This `area` element has a [semantic role][] of `link` and an [accessible name][] via `alt`.
 
 ```html
 <img src="planets.gif" width="145" height="126" alt="Planets" usemap="#planetmap" />
```

Comment:
This src doesn't exist, we should add it to assets and update the src. @jkodu any idea why the tests might not have caught this?

Reply:
Sorry, I missed this one. This means we currently have the rule published and this test case is missing the source for the image map. I couldn't find the gif anywhere in the repo. You're one of the original rule authors. Do you have this image somewhere?